docs/user-guide/features.rst

```diff
@@ -33,6 +33,27 @@ Administrators can also :ref:`upload <create-errata-recipe>` their own errata to
 a repository. Please see the :doc:`recipes` documentation to learn how to
 perform these operations.
 
+Boolean dependencies
```

Comment:
How about `Boolean (rich) dependencies`?
I think it makes sense to add `rich` somewhere as well because these dependencies are sometimes referred to as that.

Reply:
ack :)